File: comments/src/main/java/de/thm/arsnova/service/comment/exception/ForbiddenException.java
package de.thm.arsnova.service.comment.exception;

import org.springframework.http.HttpStatus;
import org.springframework.web.server.ResponseStatusException;

public class ForbiddenException extends ResponseStatusException {

  public ForbiddenException() {
    super(HttpStatus.FORBIDDEN);
  }

  public ForbiddenException(final String message) {
    super(HttpStatus.FORBIDDEN, message);
  }

  public ForbiddenException(final String message, final Throwable e) {
    super(HttpStatus.FORBIDDEN, message, e);
  }
}

File: authz/src/main/kotlin/de/thm/arsnova/service/authservice/exception/ForbiddenException.kt
package de.thm.arsnova.service.authservice.exception

import org.springframework.http.HttpStatus
import org.springframework.web.server.ResponseStatusException

class ForbiddenException : ResponseStatusException {
    constructor() : super(HttpStatus.FORBIDDEN)
    constructor(reason: String?) : super(HttpStatus.FORBIDDEN, reason)
    constructor(reason: String?, cause: Throwable?) : super(HttpStatus.FORBIDDEN, reason, cause)
}
